src/rpc/rawtransaction.cpp

```diff
@@ -210,13 +210,13 @@ static RPCHelpMan getrawtransaction()
                         RPCResult::Type::OBJ, "", "",
                         {
                             {RPCResult::Type::ELISION, "", "Same output as verbosity = 1"},
-                            {RPCResult::Type::NUM, "fee", /*optional=*/true, "transaction fee in " + CURRENCY_UNIT + ", omitted if block undo data is not available"},
+                            {RPCResult::Type::NUM, "fee", "Transaction fee in " + CURRENCY_UNIT},
                             {RPCResult::Type::ARR, "vin", "",
                             {
-                                {RPCResult::Type::OBJ, "", "utxo being spent, omitted if block undo data is not available",
+                                {RPCResult::Type::OBJ, "", "utxo being spent",
                                 {
                                     {RPCResult::Type::ELISION, "", "Same output as verbosity = 1"},
-                                    {RPCResult::Type::OBJ, "prevout", /*optional=*/true, "Only if undo information is available)",
+                                    {RPCResult::Type::OBJ, "prevout", "The previous transaction",
```

Comment:
"prevout" refers to output, not a transaction. Also here, `/*optional=*/true` and `Only if undo information is available` [seem correct](https://github.com/bitcoin/bitcoin/blob/79e007d1d60b8d600479d787667d91e05a492ab4/src/core_write.cpp#L211-L228) to me, I'm open to adding a description of prevout being the previous output.

Reply:
@dougEfresh this seems unaddressed?